templates/zoom-parts/ip-links.tpl

```diff
@@ -15,6 +15,9 @@
     <a id="IPWhois-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=oq-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois-{$index}').addClass('btn-visited');">Whois</a>
     <a id="IPWhois2-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=tl-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois2-{$index}').addClass('btn-visited');">(alt)</a>
   </div>
+  <div class="btn-group">
```

Comment:
Since this is just a single button, it doesn't need to be in its own group; look at how the other single buttons are done in this file